test/zookeeperLock.mocha.js

```diff
@@ -311,19 +312,33 @@ describe('Zookeeper lock', function () {
         };
         const lock1 = new ZookeeperLock(failImmediateConfig);
         const lock2 = new ZookeeperLock(failImmediateConfig);
+        const lock3 = new ZookeeperLock(failImmediateConfig);
+        const lock4 = new ZookeeperLock(failImmediateConfig);
```

Comment:
i could, it would be kind of awkward to work with though.. the test locks the first lock first, then tries to lock all of the rest, expecting them to fail, so would be 
```locks[0].lock('test').then(() => return promise.all(locks.slice(1,4)) ..``` 
or something and then different cleanup actions are called as well. if you feel strongly about this i can change it, or just remove some of the extra locks. One proves the point which is how the test originally was. The dupes were added while I was testing for connection leaks locally, where i had a super long timeout set, and probably aren't necessary anymore.